tests/unit/framework/test_pipeline.py

```diff
@@ -508,19 +496,16 @@ def test_pipeline_metrics_number_of_errors_counts_errors_of_all_processor_metric
         self.pipeline.metrics.pipeline = [mock_metrics_one, mock_metrics_two]
         assert self.pipeline.metrics.number_of_errors == 2
 
-    def test_create_connector_adds_versions_information_to_input_connector_config(
-        self, mock_create
-    ):
-        self.pipeline._create_logger()
-        self.pipeline._create_connectors()
+    def test_setup_adds_versions_information_to_input_connector_config(self, mock_create):
+        self.pipeline._setup()
```

Comment:
This test could also be more specific by referencing the `_input` property directly instead of the `_setup` method, as the setup method doesn't really add the version information.